<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>20</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ACCOUNT SETTINGS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>User Controls</string>
    </property>
    <widget class="QPushButton" name="changepin_btn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Change Pin</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>ADMIN/BANKER CONTROLS</string>
    </property>
    <widget class="QPushButton" name="convert_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CONVERT ALL TO AVAILABLE BALANCE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="verify_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>VERIFY ACCOUNT</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
